ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -250,8 +250,12 @@ const Annotations = React.memo(({ variant }) => {
           size="large"
           trigger={
             <ButtonLink size={svType && 'big'}>
-              {svType || mainTranscript.majorConsequence.replace(/_/g, ' ')}
-              {svType && svTypeDetail && `:${svTypeDetail}`}
+              {svType ? (SVTYPE_LOOKUP[svType] || svType) : mainTranscript.majorConsequence.replace(/_/g, ' ')}
+              {svType && svTypeDetail && SVTYPE_DETAILS[svType] &&
```

Comment:
similar to above, if there is ever `svTypeDetail` that is somehow missing from `SVTYPE_DETAILS` we still want to show it, so this should only be conditional on `svTypeDetail` and then for the popup content we want to do something like `(SVTYPE_DETAILS[svType] || {})[svTypeDetail] || svTypeDetail`